packages/utils/src/lib/transform.ts

```diff
@@ -156,3 +157,56 @@ export function toOrdinal(value: number): string {
   return `${value}th`;
 }
 /* eslint-enable no-magic-numbers */
+
+export class AuditsNotImplementedError extends Error {
+  constructor(list: { slug: string }[], auditSlugs: string[]) {
+    super(
+      `audits: "${auditSlugs
+        .filter(slug => !list.some(a => a.slug === slug))
+        .join(', ')}" not implemented`,
+    );
+  }
+}
+
+export function filterByAuditSlug<
```

Comment:
I think the common logic of these two functions could be extracted into a helper filter function which would accept just an array audit slugs and filter out those that aren't in `onlyAudits`. This function could then be called on mapped array that both `filterByAuditSlug` or `filterBySlug` can provide.

The unit tests would then test this core logic instead of duplicating use cases for each function separately.